examples/classification/4-tutorials/plot_main-tutorial-classification.py

```diff
@@ -0,0 +1,288 @@
+"""
+===========================
+Tutorial for classification
+===========================
+
+
+In this tutorial, we compare the prediction sets estimated by the conformal
+methods implemented in MAPIE on a toy two-dimensional dataset.
+
+Throughout this tutorial, we will answer the following questions:
+
+- How does the number of classes in the prediction sets vary according to
+  the significance level ?
```

Comment:
varies

Reply:
No, since there is a "does" in the sentence.